collect_app/src/main/res/layout/storage_migration_dialog.xml

```diff
@@ -0,0 +1,137 @@
+
+            <TextView
+                android:id="@+id/messageText3"
+                style="@style/TextAppearance.Collect.Body1"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/margin_small"
+                android:text="@string/storage_migration_dialog_message3" />
+
+            <TextView
+                android:id="@+id/moreDetailsButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/storage_migration_more_details"
+                android:textColor="?colorAccent"
+                android:textStyle="bold"
+                android:textAllCaps="true"
+                android:paddingTop="@dimen/margin_standard" />
+
+            <TextView
+                android:id="@+id/errorText"
+                style="@style/TextAppearance.Collect.Headline6"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/margin_standard"
+                android:visibility="gone"/>
+
+            <LinearLayout
+                android:id="@+id/progressBar"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:orientation="vertical"
+                android:gravity="center"
+                android:visibility="gone">
+
+                <ProgressBar
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content">
+                </ProgressBar>
+
+                <TextView
+                    style="@style/TextAppearance.Collect.Body1"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="@string/storage_migration_progress_msg"
+                    android:layout_marginTop="@dimen/margin_small"
+                    android:gravity="center_horizontal"/>
+            </LinearLayout>
+        </LinearLayout>
+
+        <FrameLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_alignParentBottom="true">
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="vertical">
+
+                <ImageView
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:src="@drawable/shadow_up" />
+
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:orientation="horizontal"
+                    android:gravity="end">
+
+                    <Button
```

Comment:
Again lets do `MaterialButtons` for these and see if we can get rid of the `bold` and `textAllCaps`

Reply:
Done, but bold is still needed.